eng/pipelines/linux.yml

```diff
@@ -134,35 +134,60 @@ stages:
             - linuxArm64Queues: \(Ubuntu.1604.Arm64.Open\)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-a45aeeb-20190620160300
 
       # Legs without helix testing
-      # Only run this leg in PRs.
-      - ${{ if and(eq(parameters.fullMatrix, 'false'), and(ne(parameters.testScope, 'outerloop'), ne(parameters.testScope, 'all'))) }}:
+      # Don't run these legs in outerloop builds
+      - ${{ if and(ne(parameters.testScope, 'outerloop'), ne(parameters.testScope, 'all')) }}:
         - job: LinuxNoTest
           displayName: Build
           strategy:
             matrix:
-              arm_Debug:
-                _BuildConfig: Debug
-                _architecture: arm
-                _framework: netcoreapp
-                _buildExtraArguments: /p:RuntimeOS=ubuntu.16.04 -warnAsError false
-                _buildScriptPrefix: 'ROOTFS_DIR=/crossrootfs/arm '
-                _dockerContainer: ubuntu_1604_arm_cross_container
-
-              musl_arm64_Debug:
-                _BuildConfig: Debug
-                _architecture: arm64
-                _framework: netcoreapp
-                _dockerContainer: alpine_37_arm64_container
-                _buildScriptPrefix: 'ROOTFS_DIR=/crossrootfs/arm64 '
-                _buildExtraArguments: -warnAsError false /p:BuildNativeCompiler=--clang5.0 /p:RuntimeOS=linux-musl
-
-              arm64_Debug:
-                _BuildConfig: Debug
-                _architecture: arm64
-                _framework: netcoreapp
-                _dockerContainer: ubuntu_1604_arm64_cross_container
-                _buildScriptPrefix: 'ROOTFS_DIR=/crossrootfs/arm64 '
-                _buildExtraArguments: --warnAsError false
+              # Run these legs on PRs and official builds, but not on CI builds
+              ${{ if or(eq(parameters.fullMatrix, 'false'), eq(parameters.isOfficialBuild, 'true')) }}:
```

Comment:
This doesn't make sense. We don't run tests on official builds.

Reply:
OK I was confused becaues you are adding a helixQueue below.

Reply:
Do we really want to run this in PRs? Why don't we want to run this as part of rolling builds?

Reply:
Rolling builds always need to be a super-set of PR builds. That's how we measure the CI success rate of PR builds in the CI council.

Reply:
/end

Reply:
> OK I was confused becaues you are adding a helixQueue below.

Yes that was a mistake from copy and paste 😄 -- I will remove it.

> Do we really want to run this in PRs? Why don't we want to run this as part of rolling builds?

Yes, we want this build to be guarded as part of PR builds as the native pieces are different and use a special toolset, we want to catch any issues early. 

I thought of not running them on rolling builds cause it didn't run any tests (yet) and we seem to run tests for all legs in rolling builds, if we're OK with having this leg be part of rolling builds I don't object and can fix that.

Reply:
Yes please add it to rolling builds as well.

Reply:
Done.